minoc/src/main/java/org/opencadc/minoc/GetAction.java

```diff
@@ -99,12 +113,30 @@ public void doAction() throws Exception {
         
         Artifact artifact = getArtifact(artifactURI);
         HeadAction.setHeaders(artifact, syncOutput);
-        
+
+        final List<String> requestedSubs = syncInput.getParameters(SodaParamValidator.SUB);
```

Comment:
I didn't think you would need to do this but SyncInput doesn't expose the whole param map. I had planned on something like 
```
List<ExtensionSlice> slices = SODA_PARAM_VALIDATOR.validateSUB(syncInput.getParameterMap());
```
because this is going to get ugly when you also need to do SODA and you'll want to use 
`SODA_PARAM_VALIDATOR.validateAllShapes(Map)` without having to know all the params that carry shape values.... the idea with the validators is that you let them look through the whole map and pick out the things they know about. ugh.

Ok  for now, but I'll create an issue to add SyncInput.getParameterMap() and use it here